specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -214,15 +229,16 @@ export function QueryResults(props: QueryResultsProps): JSX.Element {
         typeof fetchResults === 'function' &&
         visibleFieldSpecs.length > 0 ? (
           <>
-            {hasPermission('/record/merge', 'update') &&
-              hasTablePermission(table.name, 'update') && (
-                <RecordMergingLink
-                  selectedRows={selectedRows}
-                  table={table}
-                  onDeleted={handleDelete}
-                  onMerged={handleReRun}
-                />
-              )}
+            {(table.name === 'PaleoContext' && canMergePaleoContext) ||
+            (table.name === 'CollectingEvent' && canMergeCollectingEvent) ||
```

Comment:
when you define canMergePaleoContext or canMergeCollectingEvent you already check for table.name so you don't need to do so here

Reply:
right!